### ISSUE TITLE
Performance on Qwen2.5-7B-Instruct

### ISSUE THREAD

**Issue by kyre-99** [simplescaling/s1#132]
Thank you for your work. I tried training a 7B-sized model directly using LoRa, but despite the training loss decreasing well, it didn't perform well on the test set, even worse than the base model. I tried a simple transfer from gsm8k to aime24, but the results were not good. Why is this?

**Comment by Muennighoff** [comment]
Hmm I think GSM8K is probably not challenging enough; did you try our s1K dataset?
